sites/kit.svelte.dev/src/routes/home/Features.svelte

```diff
@@ -5,38 +5,39 @@
 <Section --background="var(--sk-back-1)">
 	<div class="container">
 		<div class="features">
-			<h2><span>features? we got 'em.</span></h2>
+			<h2><span>des fonctionnalités ? on a ça en stock.</span></h2>
 			<p class="wall">
-				Mix and match <strong>prerendered</strong> pages for maximum performance with dynamic
-				<strong>server-side rendering</strong> for maximum flexibility. Turn your app into a
-				client-rendered
-				<strong>PWA</strong> with a single line of code, for the whole thing or just one page. Use
-				accessible <strong>client-side routing</strong> with automatic
-				<strong>preloading</strong> for slick, instantaneous navigation that doesn't reload your
-				entire page (and your analytics, and all that other junk). Protect your users with automatic
-				<strong>CSRF protection</strong>
-				and easy-to-use
-				<strong>Content Security Policy</strong> configuration. Keep your secrets to yourself with
-				advanced <strong>environment variable</strong> handling. Handle errors gracefully and
-				<strong>securely</strong>. Load data
-				<strong>directly from your database</strong>
-				and connect your back end to your front end with <strong>type-safe</strong> data loading and
-				built-in <strong>form actions</strong>
-				that work with or without JavaScript. <strong>Co-exist</strong> with other client-side
-				routing frameworks on the same page. Add service workers for <strong>offline</strong>
-				support. Generate <strong>AMP-compliant</strong>
-				pages if you really have to. Build complex UIs with unusually powerful
-				<strong>filesystem-based routes</strong>. Nested layouts? Duh. Learn
-				<strong>web standards</strong>
-				that work across environments. Integrate with <strong>Tailwind</strong>
-				and <strong>Playwright</strong> and <strong>Vitest</strong> and <strong>Storybook</strong>
-				and, well, whatever you want. Build <strong>libraries</strong> as well as apps.
-				<strong>Deploy anywhere</strong> with adapters.
+				Mélangez des pages <strong>pré-rendues</strong> pour une perfomance optimale avec du
+				<strong>rendu dynamique sur le serveur</strong>
+				pour une flexibilité maximale. Transformez votre application en une
+				<strong>PWA</strong> rendue côté client avec une seule ligne de code, pour toute votre
+				application ou une seule page. Utilisez un <strong>routing côté client</strong> accessible
```

Comment:
```suggestion
				application ou seulement une seule page. Utilisez un <strong>routeur côté client</strong> accessible
```
je crois qu'on a mis routing partout mais je préfeur routeur :s